aws/sdk-codegen/src/main/kotlin/software/amazon/smithy/rustsdk/AwsFluentClientDecorator.kt

```diff
@@ -223,6 +223,7 @@ private fun renderCustomizableOperationSendMethod(
     val combinedGenerics = operationGenerics + handleGenerics
 
     val codegenScope = arrayOf(
+        *RuntimeType.preludeScope,
```

Comment:
I wonder if we should just merge this scope directly in `rustTemplate` and friends.

Reply:
I definitely thought about it. That and deleting `rust()` and renaming `rustTemplate()` to `rust()`. I'd like to get rid of the old style `#T` formatters entirely to tidy things up. All of this is quite a bit of work, and I wanted to get this initial fix out first.